TFLite/src/common/Storage.py

```diff
@@ -0,0 +1,208 @@
+import os
+from queue import Queue
+from google.cloud import spanner
+
+# Module to store Graph data into database
+# Node object attributes are stored in 'Operators' table
+# Edge object attributes are stored in 'Tensors' table
+# Graph object attributes are stored in 'Models' table
+
+# The attribute names in each case must correspond to a column name in 
+# the database. If any attribute is added in the Class files, 
+# the Storage class will automatically try to add that attribute to the 
+# database the next time it is run with any model
+
+# If an attribute is added to the class file and not to the database,
+# this will throw and error unless explicitly changed
+
+# Spanner allows only 20000 mutations per commit, so _load_operators
+# and _load_tensors implement batching
+class Storage:
+
+    # Initialize spanner client with instance and database
```

Comment:
It's preferable to use docstrings for details about the class and class methods. But we can keep it like this for now. All future changes you can handle them accordingly.

Reply:
Added docstrings